app/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'com.google.gms.google-services'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
+apply from: "$project.rootDir/jacoco.gradle"
```

Comment:
please enable test coverage for the app module.

Reply:
Test coverage is enabled here. Please take a look at [this](https://github.com/oppia/oppia-android/pull/2730#issuecomment-792005962)